### DISCUSSION TITLE
First time assistance

### DISCUSSION THREAD

**Discussion by ABakedWaffle** [BadDogSkyrim/PyNifly#234] [Q&A] [answered]
I've never used Blender before, and I think I've followed the instructions (https://docs.blender.org/manual/en/latest/editors/preferences/addons.html) but it wasn't very specific about *how* to install add-ons, so I added the files under a "Script Directory" the way it told me to, and the "Nifly" add-on is there and enabled, but it's not there under "import".... So I'm not sure what I'm supposed to do, here.

**Comment by BadDogSkyrim** [comment, marked as answer]
Nope, not like that. Take that shit out.

Download the zip file that's attached to each "release" here. 

Then before Blender 4.2, Edit -> Preferences -> Add-Ons -> Install..., then select the zip file you just downloaded.

4.2 completely buggered the system. Now, Add-ons -> tiny little down-arrow in the upper right corner -> install from disk. Select the zip.

**Reply by ABakedWaffle** [reply]
Thank you, will do!
